all.json

```diff
@@ -1955,8 +1955,8 @@
         "region_id": 2,
         "names": {
           "ar": null,
```

Comment:
It would be very great if you add the Arabic translation to this file too

Reply:
ima be fixin all the null stuff cz there r so many lol